pyirf/cuts.py

```diff
@@ -44,9 +92,14 @@ def calculate_percentile_cut(
     min_events: int
         Bins with less events than this number are replaced with ``fill_value``
     """
+
+    if weights is None:
```

Comment:
I'd prefer it (and it's probably also faster) if the `weights=None` case would use the existing code instead of the `weighted_quantile` method with weights=1.